lettergram/DataProfiler
# Updated Model (#53)

* updated version and readme

* fixed spelling

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* Update README.md

* updated model & tests

* updated tests to be more robust

* Delete #_about.py#

* updated integration test to ensure dict is consistent as expected

* updated integration test to ensure dict is consistent as expected

* add support for discerning header when datetimes available

* updated readme

* drivers license
